dht.go

```diff
@@ -179,9 +189,26 @@ func NewDHTClient(ctx context.Context, h host.Host, dstore ds.Batching) *IpfsDHT
 	return dht
 }
 
-func makeDHT(ctx context.Context, h host.Host, dstore ds.Batching, protocols []protocol.ID, bucketSize int) *IpfsDHT {
+func makeRoutingTable(h host.Host, cfg *opts.Options) (*kb.RoutingTable, error) {
 	self := kb.ConvertPeerID(h.ID())
-	rt := kb.NewRoutingTable(bucketSize, self, time.Minute, h.Peerstore())
+	// construct the routing table with a peer validation function
+	pvLogger := logging.Logger("RT peer validation")
```

Comment:
should be global

(and should have a name like "dht/rt")

Reply:
This is done.